### ISSUE TITLE
Write script that divides training to validation set + training set, according to speakers and creates egs file.

### ISSUE THREAD

**Issue by mosem** [mosem/denoiser#34]
log the different files to a table, according to speakers.
